src/lib/transformers/tokend.js

```diff
@@ -189,6 +173,12 @@ class TokendTransformer {
         transformedProperties = transformedProperties.mergeDeep(value);
       });
 
+      Object.keys(this._cache)
```

Comment:
This is going to sound dumb but can you put some comments around this ejection strategy? 

The approach should be fine for the size of the property-set we have but I can totally see this chugging if the cache gets large enough and it would be useful to capture the reasoning for the approach now in case we need to revisit down the road.

Reply:
It is not dumb at all! I added a comment, let me know if the comment is good enough.